validator_client/src/block_service.rs

```diff
@@ -422,7 +552,7 @@ impl<T: SlotClock + 'static, E: EthSpec> BlockService<T, E> {
                 return Ok(());
             }
             Err(e) => {
-                return Err(BlockError::Recoverable(format!(
+                return Err(BlockError::Irrecoverable(format!(
```

Comment:
Same here.

Reply:
Done in https://github.com/sigp/lighthouse/pull/4813/commits/f579a043067eec26072baba2bf3a8505c33f8159